packages/synapse-react-client/src/components/CreateTableViewWizard/README.md

```diff
@@ -0,0 +1,25 @@
+The following Mermaid diagram defines the general workflow of the Create Table Wizard:
+
+```mermaid
```

Comment:
VSCode (and maybe GitHub once merged?) will render this properly

<img width="910" alt="image" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/5e3191d8-bccb-4a00-aa97-072667ad127e">